static/templates/stream_sidebar_actions.hbs

```diff
@@ -14,6 +14,12 @@
     <hr>
 
     {{! tabindex="0" Makes anchor tag focusable. Needed for keyboard support. }}
+    <li>
+        <a tabindex="0" class="new_topic_stream">
+            <i class="fa fa-plus" aria-hidden="true"></i>
+            {{t "New topic" }}
+        </a>
+    </li>
```

Comment:
This one should sit lower in the list -- maybe just before "Unsubscribe".

Reply:
Sure, no worries. I think both the positions are fine.